doc/classes/EditorSettings.xml

```diff
@@ -717,9 +717,21 @@
 		<member name="filesystem/quick_open_dialog/default_display_mode" type="int" setter="" getter="">
 			If set to [code]Adaptive[/code], the dialog opens in list view or grid view depending on the requested type. If set to [code]Last Used[/code], the display mode will always open the way you last used it.
 		</member>
+		<member name="filesystem/quick_open_dialog/enable_fuzzy_matching" type="bool" setter="" getter="">
+			If [code]true[/code], fuzzy matching of search tokens is allowed.
+		</member>
 		<member name="filesystem/quick_open_dialog/include_addons" type="bool" setter="" getter="">
 			If [code]true[/code], results will include files located in the [code]addons[/code] folder.
 		</member>
+		<member name="filesystem/quick_open_dialog/max_fuzzy_misses" type="int" setter="" getter="">
+			The number of allowed character misses in a match, if fuzzy matching is enabled.
```

Comment:
A more detailed explanation would be useful.

Reply:
By "more detailed" I meant explaining what's "character misses", as it's not immediately clear.

Reply:
Oh whoops, I had reworded this and added an example but I had accidentally nuked the commit